src/hooks/useOnPressHandlers.ts

```diff
@@ -1,21 +1,28 @@
-const useOnPressHandlers = ({navigation}) => {
-  const onPressLogIn = () => {
+import { REGISTER_ROUTE, LOGIN_ROUTE } from '../constants/paths';
+
+const useOnPressHandlers = () => {
+  const onPressLogIn = (navigation, data) => {
```

Comment:
To samo bedzie sie dzialo przy home buttonie na TapBarze, osobny handler czy rename tego?

Reply:
Jak dla mnie będzie osobny handler - ten onPressLogIn dostaje dane z hookForma i nie dobrze byłoby mieszać to ze zwykłym reroutingiem.

Osobny handler w postaci `onTapBarHomePress` czy coś w tym stylu, wiesz, taka seria tap/top barowych handlerów. Potem z tego hooka łatwo będzie wyciągnąć odpowiednie callbacki do tych funkcji i będzie elegancko.

@PanStick